### DISCUSSION TITLE
2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%80%EF%BC%89Hexo-%E6%A1%86%E6%9E%B6/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#6] [General]
# 2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%80%EF%BC%89Hexo-%E6%A1%86%E6%9E%B6/

先安装 Node.js 因为Hexo依赖于Node.js运行，所以我们需要先安装Node.js 前往官网下载安装长期维护版即可，如果使用最新版本的软件时出现问题，可以尝试我使用的版本  安装完成后，按下 Windows + R 打开运行，输入CMD并回车，打开命令提示符 依次输入下面的命令查询版本号

https://nickxu.me/2022/02/13/Hexo%20+%20Butterfly%20%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E4%B8%80%EF%BC%89Hexo-%E6%A1%86%E6%9E%B6/

**Comment by illyber** [comment]
感谢分享
